README.md

```diff
@@ -90,6 +111,24 @@ Notation styles are allowed to be combined with other options:
 // ==> 3.00E8 m/s
 ```
 
+### Feature Detection: Notation
+
+Check for the notation in `resolvedOptions()`:
+
+```javascript
+/**
+ * Returns an Intl.NumberFormat if the notation is supported,
+ * or null if unsupported.
+ */
+function getIntlNumberFormatWithNotation(notation) {
+  let numberFormat = new Intl.NumberFormat("en-US", { notation });
+  if (numberFormat.resolvedOptions().notation !== notation) {
```

Comment:
nit:

```suggestion
  if (numberFormat.resolvedOptions().hasOwnProperty('notation') {
    return numberFormat;
  }
  return null;
```

This becomes a positive if condition and it's a bit more strict to avoid prototype chains.